pkg/apis/admissionregistration/fuzzer/fuzzer.go

```diff
@@ -98,5 +98,12 @@ var Funcs = func(codecs runtimeserializer.CodecFactory) []interface{} {
 				obj.MatchPolicy = &m
 			}
 		},
+		func(obj *admissionregistration.ParamRef, c fuzz.Continue) {
+			c.FuzzNoCustom(obj)
+			if obj.ParameterNotFoundAction == nil {
+				a := admissionregistration.AllowAction
```

Comment:
Don't set default as allow for ParameterNotFoundAction

Reply:
Need to keep this because v1alpha1 has a default value for `ParameterNotFoundAction`, but for the internal type being fuzzed here, it is required and not defaulted. So if internal version has `parameterNotFoundAction` unset, it will be defaulted into `Deny` upon conversion to `v1alpha1`, which does not round-trip and fails our tests.

Reply:
This made me realize the for internal it does not make sense to have required parameterNotFoundAction be a pointer. After changing it to a pointer I was able to remove this part of the fuzzer.